Example/Program.cs

```diff
@@ -0,0 +1,142 @@
+            logger.LogInformation("Querying FOMA for notifications");
+            var notificationResult = await fomaSdk.GetNotifications(fulfillerId);
+            logger.LogInformation($"Found {notificationResult.TotalCount} notifications.");
+
+            // loop through all notifications
+            foreach (var notification in notificationResult.Notifications)
+            {
+                logger.LogInformation($"Found notification {notification.NotificationId} of type {notification.Type}.");
+                if (notification.Type == "OrderRequest")
+                {
+                    try
+                    {
+                        await ImportOrder(notification);
+                        await AcceptNotification(notification);
+                    }
+                    catch (Exception ex)
+                    {
+                        logger.LogError(ex, "Error while importing order for notification {NotificationId}", notification.NotificationId);
+                    }
+                }
+            }
+        }
+
+        private async Task AcceptNotification(NotificationDto notification, NotificationAcceptRejectDto acceptRejectDetails = null)
+        {
+            logger.LogInformation($"Accepting notification {notification.NotificationId}");
+            await fomaSdk.SendData<NotificationAcceptRejectDto>(notification.Links[LinkRels.Accept].Href);
+        }
+
+        private async Task RejectNotification(NotificationDto notification, NotificationAcceptRejectDto acceptRejectDetails = null)
+        {
+            logger.LogWarning($"Rejecting notification {notification.NotificationId}");
+            await fomaSdk.SendData<NotificationAcceptRejectDto>(notification.Links[LinkRels.Accept].Href);
+        }
+
+        private async Task ImportOrder(NotificationDto notification)
+        {
+            logger.LogInformation($"Processing notification for order {notification.Order.OrderId}");
+            foreach (var nItem in notification.Items)
+            {
+                logger.LogInformation($"Downloading details for item {nItem.ItemId}: item details");
+                var item = await fomaSdk.GetData<ItemDto>(nItem.Links[LinkRels.Self].Href);
+
+                logger.LogInformation($"Downloading details for item {nItem.ItemId}: manufacturing details");
+                var manufacturingDetails = await fomaSdk.GetData<ManufacturingDetailDto>(item.Links[LinkRels.ManufacturingDetails].Href);
+
+                logger.LogInformation($"Downloading details for item {nItem.ItemId}: order details");
+                var order = await fomaSdk.GetData<OrderDto>(item.Order.Links[LinkRels.Self].Href);
+
+                logger.LogInformation($"Downloading details for item {nItem.ItemId}: artwork / document");
+                using (var response = await fomaSdk.Download(item.Links[LinkRels.Document].Href))
+                {
+                    var suggestedFilename = response.Content.Headers.ContentDisposition.FileName;

```

Comment:
For some reason, the filename is in extra double quotes, causing the extension to be e.g. `gif"`.

![screen shot 2018-07-05 at 15 06 49](https://user-images.githubusercontent.com/9152385/42325015-42564476-8065-11e8-8a99-1197b8aee788.png)

According to https://tools.ietf.org/html/rfc6266#section-4.3 usage of `FileNameStart` is recommended over `FileName`.